calcom/cal.com
# feat: Expose booking created at in v2 API response

## What does this PR do?

Expose booking created at in v2 API response

- Fixes #10670 

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
Run the v2 API and checking the response for `/v2/bookings` and `/v2/bookings/{uid}`, you should see the `createdAt` in the response
